src/node/desktop/src/ui/README.md

```diff
@@ -0,0 +1,72 @@
+
+# RStudio Electron UI
+
+This README documents the structure of custom widgets used in RStudio.
+
+For example, consider a widget called 'clock'. The following files should be created:
+
+```
+widgets/clock.ts            <required>
+widgets/clock/ui.html       <required>
```

Comment:
Let's keep the `html` file outside the `src` tree, probably in a `resources` folder (for example, `src/node/desktop/resources/widgets`). Then refer to it using the `appState().resourcesPath().completePath()` technique mentioned elsewhere.

I'll probably end up moving the existing `html` folder into this as well (`src/node/desktop/resources/html`).